packages/pieces/apps/src/lib/generatebanners/index.ts

```diff
@@ -0,0 +1,12 @@
+import { createPiece } from '@activepieces/framework';
+import { renderTemplate } from './actions/renderTemplate.action';
+
+export const generatebanners = createPiece({
+  name: 'generatebanners',
+  logoUrl: 'https://www.generatebanners.com/logo-transparent-512.png',
```

Comment:
Uploaded, can you please change it to

`https://cdn.activepieces.com/pieces/generatebanners.png`

Reply:
Fixed, thanks!